lib/repository.ts

```
@@ -1,14 +1,14 @@
-import git, { GitAuth, Errors } from 'isomorphic-git';
-import * as http from 'isomorphic-git/http/node';
-import * as path from 'path';
-import * as fs from 'fs';
-import { fsExists, getContentPath } from './lib';
+import git, { GitAuth, Errors } from "isomorphic-git";
+import * as http from "isomorphic-git/http/node";
+import * as path from "path";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)